flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/ForStFlinkFileSystem.java

```diff
@@ -61,6 +70,16 @@ public static FileSystem get(URI uri) throws IOException {
         return new ForStFlinkFileSystem(FileSystem.get(uri));
     }
 
+    /**
+     * Setup local base path for corresponding remote base path.
+     *
+     * @param remoteBasePath the remote base path.
+     * @param localBasePath the local base path.
+     */
+    public static void setupLocalBasePath(String remoteBasePath, String localBasePath) {
+        remoteLocalMapping.put(remoteBasePath, localBasePath);
```

Comment:
Is there any way removing the map entry? I'm worried this would keep growing.

Reply:
I add `unregisterLocalBasePath`, and call it when `keyedStateBackend.dispose`